src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer bulldozer1 = new Bulldozer(4.5, 5);
+    private Bulldozer bulldozer2 = new Bulldozer(5.5, 6.5);
+
+    private List<Bulldozer> bulldozers = new ArrayList<>();

```

Comment:
It's bad practice to use numbers in variable names. And lets make this variables local (same for other clases)